modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/RelatedObjectEntryResourceImpl.java

```diff
@@ -54,6 +54,51 @@
 public class RelatedObjectEntryResourceImpl
 	extends BaseRelatedObjectEntryResourceImpl {
 
+	@Override
+	public void deleteObjectRelationshipMappingTableValues(
+			String previousPath, Long objectEntryId,
+			String objectRelationshipName, Long relatedObjectEntryId)
+		throws Exception {
+
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153324"))) {
+			throw new NotFoundException();
+		}
+
+		ObjectDefinition systemObjectDefinition = _getSystemObjectDefinition(
+			previousPath);
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.
+				getObjectRelationshipByObjectDefinitionId(
+					systemObjectDefinition.getObjectDefinitionId(),
+					objectRelationshipName);
+
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.getObjectDefinition(
+				objectRelationship.getObjectDefinitionId1());
+
+		_objectRelationshipService.deleteObjectRelationshipMappingTableValues(
+			objectRelationship.getObjectRelationshipId(),
+			_getPrimaryKey1(
+				objectDefinition, objectEntryId, relatedObjectEntryId,
+				systemObjectDefinition),
+			_getPrimaryKey2(
+				objectDefinition, objectEntryId, relatedObjectEntryId,
+				systemObjectDefinition));
+
+		ObjectEntryManager objectEntryManager =
+			_objectEntryManagerTracker.getObjectEntryManager(
+				systemObjectDefinition.getStorageType());
+
+		ObjectDefinition relatedObjectDefinition = _getRelatedObjectDefinition(
+			systemObjectDefinition, objectRelationship);
+
+		objectEntryManager.getObjectEntry(
```

Comment:
Why do we get the object entry?

Reply:
Just to check if the `ObjectId` is related with the ObjectDefinition that has a relationship with the System Object

Reply:
If the `ObjectId` is not related to this ObjectDefinition, it throws a 404 status code

Reply:
I think the code should be returned before invoking the method to delete the relationship as we would already know that the object entry does not exist.

I made some tests with the PUT method and I think we should change it too (maybe in other PR). Maybe we should discuss it internally

Reply:
What do you think @4lejandrito ?